hack/ovn-kind-cno.sh

```diff
@@ -120,6 +128,9 @@ kubectl create -f https://raw.githubusercontent.com/openshift/api/e7fa4b871a2598
 # Create Network resource
 kubectl create -f https://raw.githubusercontent.com/openshift/api/e7fa4b871a25985ef0cc36c2fbd9f2cb4445dc9c/config/v1/0000_10_config-operator_01_network.crd.yaml
 
+# Create Infrastructure resource
+kubectl create -f https://raw.githubusercontent.com/openshift/api/e7fa4b871a25985ef0cc36c2fbd9f2cb4445dc9c/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml
```

Comment:
why are these manifests pinned? should they use master too? https://raw.githubusercontent.com/openshift/api/master/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml
how do we know the relation between openshift/api and openshift/cluster-network-operator?